qemu/tests/sev_basic_config.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sev basic test on Milan and above host:
+    1. Check host sev capability
+    2. Boot sev VM
+    3. Verify sev enabled in guest
+    4. Check sev qmp cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sev test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
+    sev_module_path = params["sev_module_path"]
+    if os.path.exists(sev_module_path):
+        f = open(sev_module_path, "r")
+        output = f.read().strip()
+        f.close()
```

Comment:
A better way to read a file:

`
with open(sev_module_path, 'r') as fd:
    output = fd.read().strip()
`

But yours is still OK for me, just FYI

Reply:
Agree with Zhenchao, "with open" statement will close the file automatically, no need f.close() in the end.